src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+openapi: 3.0.0
```

Comment:
I suggest splitting this into separate files for better readability. However, this is not supported by `connexion` out-of-the box. It can be [achieved quite easily though](https://github.com/zalando/connexion/issues/254#issuecomment-504699959).

Reply:
I played with that for a while and I don't think this is worth it. We can basically split it just into `paths` and `components`, which doesn't much help. Also, this breaks validation and suggestions in IDE.